libs/apriltag/CMakeLists.txt

```diff
@@ -0,0 +1,298 @@
+    target_link_libraries(${PROJECT_NAME}-link-libs INTERFACE Threads::Threads m)
+    target_link_libraries(${PROJECT_NAME}-utils-link-libs INTERFACE m)
+endif()
+
+# Detector Library
+set(HEADERS_DETECTOR_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include/apriltag)
+
+set(HEADERS_DETECTOR
+    ${HEADERS_DETECTOR_DIR}/apriltag.h
+    ${HEADERS_DETECTOR_DIR}/apriltag_pose.h
+    ${HEADERS_DETECTOR_DIR}/common/g2d.h
+    ${HEADERS_DETECTOR_DIR}/common/homography.h
+    ${HEADERS_DETECTOR_DIR}/common/matd.h
+    ${HEADERS_DETECTOR_DIR}/common/postscript_utils.h
+    ${HEADERS_DETECTOR_DIR}/common/pthreads_cross.h
+    ${HEADERS_DETECTOR_DIR}/common/svd22.h
+    ${HEADERS_DETECTOR_DIR}/common/timeprofile.h
+    ${HEADERS_DETECTOR_DIR}/common/unionfind.h
+    ${HEADERS_DETECTOR_DIR}/common/workerpool.h
+    ${HEADERS_DETECTOR_DIR}/common/zmaxheap.h
+)
+
+SET(SRC_DETECTOR
+    src/apriltag.c 
+    src/apriltag_pose.c 
+    src/apriltag_quad_thresh.c
+    src/common/g2d.c
+    src/common/homography.c
+    src/common/matd.c
+    src/common/pthreads_cross.cpp
+    src/common/svd22.c
+    src/common/workerpool.c
+    src/common/zmaxheap.c
+    ${HEADERS_DETECTOR}
+)
+
+add_library(${PROJECT_NAME}-detector 
+    ${SRC_COMMON}
+    ${SRC_DETECTOR}
+    ${HEADERS_COMMON}
+    ${HEADERS_DETECTOR}
+)
+
+target_link_libraries(${PROJECT_NAME}-detector 
+    PUBLIC
+        ${PROJECT_NAME}-include 
+        ${PROJECT_NAME}-link-libs
+)
+
+set_apriltag_lib_property_defaults(${PROJECT_NAME}-detector)
+set_apriltag_named_exports_only(${PROJECT_NAME}-detector)
+set_target_properties(${PROJECT_NAME}-detector PROPERTIES EXPORT_NAME detector)
+# auto generate headers used for all libs here - ensure this call after
+# set_apriltag_named_exports_only() as that fn overrides DEFINE_SYMBOL
+generate_export_header(${PROJECT_NAME}-detector BASE_NAME apriltag)
+
+# Tag Family Libraries
+set(TAG_FAMILIES
+    tag16h5
+    tag25h9
+    tag36h10
+    tag36h11
+    tagCircle21h7
+    tagCircle49h12
+    tagCustom48h12
+    tagStandard41h12
+    tagStandard52h13
+)
+
+set(TARGETS_TAGS "")
+set(HEADERS_TAGS "")
+set(SRC_TAGS "")
+
+foreach(TAG_FAMILY ${TAG_FAMILIES})
+    # for each tag family
+    set(TAG_FAMILY_HEADER ${HEADERS_DETECTOR_DIR}/${TAG_FAMILY}.h)
+    set(TAG_FAMILY_SRC src/${TAG_FAMILY}.c)
+
+    set(TAG_TARGET ${PROJECT_NAME}-${TAG_FAMILY})
+
+    # create Tag-Family object target
+    add_library(${TAG_TARGET} ${TAG_FAMILY_SRC} ${TAG_FAMILY_HEADER})
+    set_apriltag_lib_property_defaults(${TAG_TARGET})
+    set_apriltag_named_exports_only(${TAG_TARGET})
+    set_target_properties(${TAG_TARGET} PROPERTIES EXPORT_NAME ${TAG_FAMILY})
+    target_link_libraries(${TAG_TARGET} PUBLIC ${PROJECT_NAME}-include)
+
+    LIST(APPEND TARGETS_TAGS ${TAG_TARGET})
+    LIST(APPEND HEADERS_TAGS ${TAG_FAMILY_HEADER})
+    LIST(APPEND SRC_TAGS ${TAG_FAMILY_SRC})
+endforeach()
+
+# All-Tag-Families library
+add_library(${PROJECT_NAME}-tags ${SRC_TAGS} ${SRC_HEADERS})
```

Comment:
I don't see `SRC_HEADERS` defined anywhere. I think this is empty and can be removed.